reusable_workflows/check_cla/check_cla_pr.py

```diff
@@ -73,19 +73,20 @@ def create_cla_issue(self, user: str) -> GHIssue:
             ),
         )
         # replace with PENDING, once new bot has been released
```

Comment:
The comment isn't relevant anymore is it?

Reply:
Nope, I'll remove it 👍